bin/notificationSubscribedUsers.ts

```diff
@@ -0,0 +1,49 @@
+import { UserAction, UserActionType } from '../src/entity';
+import createOrGetConnection from '../src/db';
+import * as fs from 'fs';
+import { parse } from 'csv-parse';
+
+(async () => {
+  const con = await createOrGetConnection();
+  const existing = await con
+    .getRepository(UserAction)
+    .findBy({ type: UserActionType.EnableNotification });
+  const subscribed = {};
+
+  console.log('reading csv');
+  const stream = fs
+    .createReadStream('bin/onesignal_subscribed_notifications.csv')
+    .pipe(parse({ delimiter: ',', from_line: 2 }));
+
+  stream.on('error', (err) => {
+    console.log('failed to read file: ', err.message);
+  });
+
+  stream.on('data', function ([userId]) {
+    if (
+      !userId?.trim() ||
+      existing.some(({ userId: existingId }) => existingId === userId)
+    ) {
+      return;
+    }
+
+    console.log('pushing: ', userId);
+    subscribed[userId] = true;
```

Comment:
Used object, for easier filtering unique ids as users can have multiple devices from this CSV.